### DISCUSSION TITLE
2023-09-25

### DISCUSSION THREAD

**Discussion by woo-jk** [woowacourse-teams/2023-haru-study#565] [daily_meeting]
### 개인 일정
- 노아: 1시 50분 근로 팀회의
- 노아 마코 룩소: 3시 근로 회의

### 오늘의 할 일
- 6차 데모데이 방향 회의(4시)
  - 어드민 페이지
- 홍보 수단 글 및 카드뉴스 만들기

### 내일의 할 일

**Comment by aak2075** [comment]
### 체크인

10점 - 쾌적한 잠실 공기

### 체크아웃

10점 - 불타는 월요일 입니다

**Comment by nlom0218** [comment]
### 체크인

10점 - 조용하고 쾌적한 자리이네요 :)

### 체크아웃

10점 - 오늘 하루도 열심히 살았네요

**Comment by jaehee329** [comment]
### 체크인
8점 - 주말에 너무 공부를 안해버렸네요. 열심히 살게요
### 체크아웃
8점 - 할거 많네요

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 어제 그래도 미션 제출하고 자서 지금은 좀 후련한 상태입니다. 자리 너무 좋아용 ㅎㅎ

### 체크아웃
9점 - 홍보를 본격적으로 시작하니까 기분이 묘하네요. 할거 잘 해보죠.

**Comment by woosung1223** [comment]
### 체크인
8점 - 피곤하긴 한데 자리가 맘에 들긴 하네용 굿

### 체크아웃
9점 - 오늘 논의가 어느정도 잘 진행된 거 같아서 좋아요!

**Comment by woo-jk** [comment]
### 체크인
8점 - 미션해야합니다

### 체크아웃
9점 - 반응형 진행페이지 완료

**Comment by yeopto** [comment]
### 체크인

10점 - 상쾌한 월요일입니다.

### 체크아웃

10점 - 미션을 열심히 해봅시다.
